package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@evolv/javascript-sdk",
+  "version": "1.0.0",
+  "description": "This is the official JavaScript client for the Evolv Technologies optimization platform.",
+  "main": "dist/index.js",
+  "module": "src/index.js",
+  "exports": {
+    "import": "./src/index.js",
+    "require": "./dist/index.js"
+  },
+  "type": "module",
+  "homepage": "https://www.evolv.ai",
+  "bugs": {
+    "url": "https://github.com/evolv-ai/javascript-sdk/issues",
+    "email": "support@evolv.ai"
+  },
+  "directories": {
+    "doc": "./docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/evolv-ai/javascript-sdk"
+  },
+  "scripts": {
+    "prepublish": "webpack --config=webpack.prod.cjs",
+    "test": "mocha",
+    "build": "webpack --config=webpack.dev.cjs",
+    "docs": "jsdoc -c jsdocs.config.json src/index.js src/docs/README.md"
+  },
+  "author": {
+    "name": "Evolv Technologies",
+    "email": "support@evolv.ai"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "base64-arraybuffer": "^0.2.0",
+    "deepmerge": "^4.2.2"
+  },
```

When included as peer dependencies, it's also convenient to add them to `devDependencies` so that they are installed when working in the library itself.

```suggestion
  "peerDependencies": {
    "base64-arraybuffer": "^0.2.0",
    "deepmerge": "^4.2.2"
  },
```